### ISSUE TITLE
Fix the mongo connection string in INSTALLATION.md

### ISSUE THREAD

**Issue by adi-uchiha** [PalisadoesFoundation/talawa-api#1093]
**Describe the bug**
Discussion thread [Link](https://thepalisadoes-dyb6419.slack.com/archives/CSWGQ70UQ/p1677243375860119?thread_ts=1677171179.688719&cid=CSWGQ70UQ)
The steps should include to mention default development db in the connection string. 

**To Reproduce**

1.
2.
3.
4.

**Expected behavior**

**Actual behavior**
A clear and concise description of how the code performed w.r.t expectations.

**Screenshots**

**Additional details**

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by palisadoes** [comment]
@adi-uchiha please take the time to fill in the form. It helps to provide a record of why this PR was created.

The slack link will be invalid in 90 days.

We created the forms so that it would be easier for persons approving the PR to get the context immediately.

**Comment by adi-uchiha** [comment]
@palisadoes I've updated the form

**Comment by xoldyckk** [comment]
@adi-uchiha When you didn't provide the db name the server wasn't running at all? Or it was running, the data was not being created? Or you just didn't know what database the data was being stored in because there was no database name?

**Comment by adi-uchiha** [comment]
@xoldyckk The connection was successful there was no error. But no changes were made in the database. No database was created.

**Comment by xoldyckk** [comment]
> @xoldyckk The connection was successful there was no error. But no changes were made in the database. No database was created.

Was data being created when you tried creating organization or something from talawa-admin?

**Comment by adi-uchiha** [comment]
That might be happening in local mongo server. I was using Cloud. I couldn't see the 'talawa-admin' show up

**Comment by xoldyckk** [comment]
> @xoldyckk That might be happening in local mongo server. I was using Cloud. I couldn't see the 'talawa-admin' show up

There won't be any `talawa-admin` named db showing up. talawa-api uses whatever you passed in as default database. If object creation was working even without mentioning database in connection string, that means the data was stored somewhere on your mongodb atlas. IDK where tho. 

But anyway you can go ahead and update the docs.

**Comment by xoldyckk** [comment]
@adi-uchiha I just found out that mongodb uses a default database name if it's not provided. The name is `test`. You can mention this in the docs too.

**Comment by adi-uchiha** [comment]
@xoldyckk yeah we should do that. we should give readers a step to register from panel too before editing the user object. Then only the user object will appear. One new comer was stuck in this on [Slack](https://thepalisadoes-dyb6419.slack.com/archives/CSHP0N8CS/p1677400281113589) It is just a minor issue. We should provide clear steps for those who are new to this. Should I start working on this PR ?

**Comment by xoldyckk** [comment]
@adi-uchiha Already assigned you. Btw what do u mean register from panel?

**Comment by adi-uchiha** [comment]
> @adi-uchiha Already assigned you. Btw what do u mean register from panel?

The user object don't appear in the db unless we trigger a register from admin panel. Right ? This step isn't talked about.

**Comment by xoldyckk** [comment]
> > @adi-uchiha Already assigned you. Btw what do u mean register from panel?
> 
> The user object don't appear in the db unless we trigger a register from admin panel. Right ? This step isn't talked about.

Isn't that how it's supposed to work? How can you have anything in database without creating it?

**Comment by adi-uchiha** [comment]
Exactly It do not tell readers to do that step

**Comment by xoldyckk** [comment]
> @xoldyckk Exactly It do not tell readers to do that step

It shouldn't. That step should be mentioned in talawa-admin, not here.

**Comment by adi-uchiha** [comment]
@xoldyckk i was thinking about how should I do this. It would be good if I can know how you think it should be done. 
Generally new comers will go and read installation.md in talawa-api repo. But in the middle of those steps we want reader to clone the admin repo and do a register. How should I mention this ?

**Comment by xoldyckk** [comment]
@adi-uchiha There was PR for automating creation of at least one user to start working with talawa-admin. You can work on it if you want.
https://github.com/PalisadoesFoundation/talawa-api/pull/941

**Comment by palisadoes** [comment]
There is also this one: 

- https://github.com/PalisadoesFoundation/talawa-admin/issues/499

Which should really be in this repo as that is where all the work needs to be done

**Comment by adi-uchiha** [comment]
@palisadoes we should close all the related issues and open a clearly described issue in api repo. Are we planning for automation ?

**Comment by adi-uchiha** [comment]
@palisadoes Closing as not required
